docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/SDCASupportVectorMachine.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class SDCASupportVectorMachine
```

Comment:
>SDCASupportVectorMachine [](start = 17, length = 24)

This is a static class right? (Only if you happen to post another commit after this.)

Reply:
Hopefully #2548 can fix it.